### ISSUE TITLE
Cant find latest version 1.0.6 on bower

### ISSUE THREAD

**Issue by jayaram171992** [Pixabay/jQuery-autoComplete#18]
Hi,
I can't find the latest version on bower update and the bower UI is not showing your plugin. Would be great if i could bower install it.

**Comment by jayaram171992** [comment]
Thanks for the wonderful plugin. Its really good.

**Comment by SimonSteinberger** [comment]
The bower file is ready. Unfortunately, I'm working on Windows and couldn't manage to register the plugin/bower file. Is there any tutorial on how to achieve this with Windows?

**Comment by jayaram171992** [comment]
I am not sure if this is what your looking for https://ksylvest.com/posts/2013-07-23/packaging-a-jquery-plugin-with-bower. Sorry for the late reply.

**Comment by SimonSteinberger** [comment]
No, that is just a Linux tutorial on how to register bower packages. Won't help on Windows.

**Comment by jayaram171992** [comment]
As far as i thought they were the same things as long as you used Node terminal on windows. I work on a Mac so havent tried it. Will dig through and see if there is something different,

**Comment by codenut** [comment]
@SimonSteinberger, do you have git on your windows? You can add tags using git by the this command:

git tag 1.0.6
git push origin 1.0.6

Can you also do that with version 1.0.7?

Thanks!
